src/main/java/org/snomed/snowstorm/core/data/services/MultiSearchService.java

```diff
@@ -229,6 +288,42 @@ public Set<String> getAllPublishedVersionBranchPaths() {
 		return publishedVersionBranchPaths;
 	}
 	
+	public Set<String> getCurrentRefsetBranchPaths() {
+		List<Branch> allBranches = branchService.findAll();
+		Set<String> refsetBranchPaths = new HashSet<>();
+		Map<String,String> refsetToCurrentBranchPathMap = new HashMap<>();
+		//Identify the latest Refset edit branch for each refset
+		//For example:
+		//MAIN/SNOMEDCT/REFSETS/REFSET-48353439008-1676579557135/EDIT-1676579564596
+		//Is for refset 4853439008, timestamp 1676579564596
+		final Pattern p = Pattern.compile("^(.*)(REFSET-)([0-9]+)(-[0-9]+\\/EDIT-)([0-9]+)$");
+		synchronized(this) {
+			for (Branch b : allBranches) {
+				Matcher m = p.matcher(b.getPath());
+				if (m.find()) {
+					final String refset = m.group(3);
```

Comment:
Your suggestion is a good one.  Creating 'final int's  in the "getCurrentRefsetBranchPaths" method rather than 'final static int's at the top, since they're only used in this one place.

Reply:
Updates made and changes pushed.